packages/documentation/copy/en/developer/Getting_Started_Linux.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Getting started on Linux 
+layout: docs
+permalink: /docs/handbook/download
+oneline: "Quick start with Lingua Franca on Linux"
+preamble: >
+---
+
+# Getting started on Linux
+
+## Installation
+
+### Debian & Other
+
+Sadly there are currently no packaged version of the lingua-franca compiler in debian distributions so there is
+really only the manual way. But we offer prebuild binaries [here](https://github.com/lf-lang/lingua-franca/releases). 
+Please note that you need a modern version of java 11 or greater.
+
+If you want to compile the lingua-franca compiler from scratch there is a [guide](Downloading\ and\ Building.md) for that. 
+
+```
+    wget https://github.com/lf-lang/lingua-franca/releases/download/v0.2.0/lfc_0.2.0.tar.gz
+    tar xf lfc_0.2.0.tar.gz
+    export PATH="$(pwd)/lfc_0.2.0/bin/":$PATH // maybe add that to you bashrc/zshrc aliases
+    lfc ./src/yourproject.lf
+```
+
+### Arch
+
+There are packaged version of [lfc](https://aur.archlinux.org/packages/lfc-bin) and [epoch](https://aur.archlinux.org/packages/epoch-bin) the arch user repository.
+
+```bash
+    git clone https://aur.archlinux.org/packages/lfc-bin
+    cd lfc-bin
+    makepkg -si
+```
+
+### Nixos
+
+For nixos users there is a already a package since the **22.05** release. Because this is a very fast developing project
+the version is nixpkgs is always lacking behind a little bit.
+
```

Comment:
I'm confused. Is 22.05 the version number for NixOS?  Is it NixOS that rapidly developing or LF? Should this say "the version _in_ nixpkgs..."?